emission/core/wrapper/suggestion_sys.py

```diff
@@ -13,6 +13,7 @@
 import os
 import emission.net.ext_service.geocoder.nominatim as geo
 from emission.net.ext_service.geocoder.nominatim import Geocoder
```

Comment:
nit: why do you need two imports from the same module?

Reply:
I wasn't sure how to use the Geocoder from the import as geo so I used "from emission.net.ext_service.geocoder.nominatim import Geocoder"

Reply:
If you already have `import emission.net.ext_service.geocoder.nominatim as geo` then basically `geo` is a shortcut for `emission.net.ext_service.geocoder.nominatim`. so you would do `geo.Geocoder`

I prefer `import ... as` instead of `from .... ` because then if you have multiple imported geocoders (e.g. nominatim and google), you can have two geocoder objects distinguished by their prefix